packages/tokens/FigmaToCunningham.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This util script is used to convert Figma's Token Studio JSON color config to TS file.
+ *
+ * CLI usage example:
+ *  $ yarn run ts-node FigmaToCunningham.ts dark.json
```

Comment:
Yes I understand but I think it would quite pollute the repo as the real data are in src/bin/ThemeColors, you can test this if you want https://pastebin.com/bJPtWLSb